src/engraving/libmscore/note.cpp

```diff
@@ -558,7 +558,7 @@ Note::Note(Chord* ch)
     _playEvents.push_back(NoteEvent());      // add default play event
     _cachedNoteheadSym = SymId::noSym;
     _cachedSymNull = SymId::noSym;
-    _fretConflictResolveSupported = score()->styleB(Sid::fretConflictResolveSupported);
+    _fretConflictResolveSupported = !engravingConfiguration()->guitarProImportExperimental();
```

Comment:
I'm thinking that `_fretConflictResolveSupported` should maybe not be a stored member. I would prefer if `fretConflictResolveSupported` was defined like this:
```c++
bool Note::fretConflictResolveSupported() const
{
    return !engravingConfiguration()->guitarProImportExperimental();
}
```